pkg/network/netlink/consumer.go

```diff
@@ -337,6 +337,41 @@ func (c *Consumer) dumpTable(family uint8, output chan Event, ns netns.NsHandle)
 	})
 }
 
+// LoadNfConntrackKernelModule requests a dummy connection tuple from netlink conntrack which is discarded but has
+// the side effect of loading the nf_conntrack_netlink module
+func LoadNfConntrackKernelModule(ns netns.NsHandle) error {
+	sock, err := NewSocket(ns)
+	if err != nil {
+		return fmt.Errorf("could not open netlink socket for net ns %d: %w", int(ns), err)
+	}
+	defer sock.Close()
+
+	conn := netlink.NewConn(sock, sock.pid)
+	defer func() {
+		_ = conn.Close()
+	}()
+
+	dummyTupleData := []byte{0x2, 0, 0, 0, 0, 0, 0}
```

Comment:
How did you arrive at this sequence of bytes?  Could you add a comment explaining it?